spec/index.html

```diff
@@ -1174,9 +1174,13 @@ <h2>Turtle Grammar</h2>
 
   <p>A <a>Turtle document</a> is an <a data-cite="RDF12-CONCEPTS#dfn-rdf-string">RDF string</a>
     encoded in UTF-8 [[!RFC3629]].
-  Only <a data-cite="I18N-GLOSSARY#dfn-code-point">Unicode code points</a>
-  in the range <code class="codepoint">U+0000</code> to <code class="codepoint">U+10FFFF</code>
-  inclusive are allowed.
+    Only <a data-cite="I18N-GLOSSARY#dfn-scalar-value" class="lint-ignore">Unicode scalar values</a>,
+    <a data-cite="I18N-GLOSSARY#dfn-code-point">Unicode code points</a> 
+    in the range <code class="codepoint">U+0000</code> to <code class="codepoint">U+10FFFF</code>
+    inclusive, excluding the range <code class="codepoint">U+D800</code> to 
+    <code class="codepoint">U+DFFF</code> 
+    (<a data-cite="I18N-GLOSSARY#dfn-surrogate" class="lint-ignore">surrogate code points</a>),
+    are allowed.
```

Comment:
I think this sentence is redundant with the UTF-8 definition that states <q>The definition of UTF-8 prohibits encoding character numbers between U+D800 and U+DFFF</s> but it's maybe a good idea to add it anyway to make things cristal clear.

Reply:
Agreed. At one level RDF string is enough.

It would be better written as "the range U+0000 to U+D7FF and U+E000 to U+10FFFF inclusive" because Unicode writes it that way. 

A next sentence saying "no surrogate code points" is then an optional extra for clarity.